pom.xml

```diff
@@ -166,10 +170,11 @@
             <artifactId>acf-sponge</artifactId>
             <version>0.5.0-SNAPSHOT</version>
         </dependency>
+        <!-- temporary fix for https://github.com/aikar/commands/pull/216 -->
         <dependency>
-            <groupId>co.aikar</groupId>
-            <artifactId>acf-velocity</artifactId>
-            <version>0.5.0-SNAPSHOT</version>
+            <groupId>com.github.egg82</groupId>
+            <artifactId>commands</artifactId>
+            <version>fix~velocity-SNAPSHOT</version>
```

Comment:
???

Reply:
as i said its fork from jitpack, but it just was merged so i will update this pr tomorrow